pkg/git/gitlab/gitlab_client.go

```diff
@@ -53,6 +54,13 @@ func (g *GitlabClient) EnsurePaCMergeRequest(repoUrl string, d *gp.MergeRequestD
 		}
 		d.BaseBranchName = baseBranch
 	}
+	baseBranchExists, err := g.branchExist(projectPath, d.BaseBranchName)
+	if err != nil {
+		return "", err
+	}
+	if !baseBranchExists {
+		return "", boerrors.NewBuildOpError(boerrors.EGitLabBranchDoesntExist, fmt.Errorf("branch doesn't exist"))
```

Comment:
Does it make sense to include branch name in the message?

Reply:
yes